### PR TITLE
PMM-12288 fix devcontainer build and tests

### PR DESCRIPTION
**Pull request by atymchuk**
PMM-12288

Link to the Feature Build: https://pmm.cd.percona.com/job/build-dev-container/1345/console

Note: the setup command in `setup.py` is failing, so we disabled it temporarily.

### CODE REVIEW

File: .github/workflows/managed.yml [hunk 1]
```diff
@@ -9,14 +9,14 @@ on:
...
     paths-ignore:
-      - "admin/**"
-      - "agent/**"
-      - "api-tests/**"
-      - "cli-tests/**"
-      - "docs/**"
-      - "qan-api2/**"
-      - "update/**"
-      - "vmproxy/**"
+      - 'admin/**'
```
Comment (atymchuk) [inline .github/workflows/managed.yml:12]: Autoformat kicked in

**Review by BupycHuk — APPROVED** [review]

File: .devcontainer/setup.py [hunk 2]
```diff
@@ -105,7 +100,7 @@ def main():
     make_init()
 
     # do basic setup
-    setup()
+    # setup()
```
[the remark this answers is not in the document]
Comment (atymchuk) [reply .devcontainer/setup.py:103]: We can't because it's failing. I'll leave a TODO however.

File: .devcontainer/setup.py [hunk 3]
```diff
@@ -25,21 +24,17 @@ def install_packages():
...
 
     run_commands([
-        # to install man pages
-        "sed -i '/nodocs/d' /etc/yum.conf",
-
-        # reinstall with man pages
-        "yum reinstall -y yum rpm",
-
-        "yum install -y gcc git make pkgconfig glibc-static \
+        # dh-autoreconf was previously needed for installing libgit2
```
[the remark this answers is not in the document]
Comment (atymchuk) [reply .devcontainer/setup.py]: Done.

File: .devcontainer/setup.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (atymchuk) [reply .devcontainer/setup.py]: Done.

File: .devcontainer/setup.py [hunk 4]
```diff
@@ -105,7 +99,8 @@ def main():
     make_init()
 
     # do basic setup
-    setup()
+    # TODO: fix the setup and revert
+    # setup()
```
Comment (artemgavrilov) [inline .devcontainer/setup.py:103]: This will be fixed in another PR?

Comment (artemgavrilov) [reply .devcontainer/setup.py:103]: Ah, read PR description

**Review by artemgavrilov — APPROVED** [review]

File: .devcontainer/setup.py (hunk 4 above)
Comment (atymchuk) [reply .devcontainer/setup.py:103]: Yes, preferably, to let the devcontainer get built.

**Comment by atymchuk** [comment]
@percona/pmm-admins Could you please merge it?
